core/geometry/src/curve/Arc3d.ts

```diff
@@ -139,6 +139,8 @@ export class EllipticalArcApproximationOptions {
    * default quadratic. For best results, this function should be a bijection.
    * @param forcePath whether to return a [[Path]] instead of a [[Loop]] when approximating a full elliptical arc,
    * default false.
+   * * Visualization can be found at https://www.itwinjs.org/sandbox/SaeedTorabi/ArcApproximationGeneral/ and
+   *  https://www.itwinjs.org/sandbox/SaeedTorabi/ArcApproximation
```

Comment:
```suggestion
   *  https://www.itwinjs.org/sandbox/SaeedTorabi/ArcApproximation/
```

Reply:
This suggestion is so that the two URLs are consistent with each other. I think they both work fine with or without the trailing slash.

Reply:
Will make them consistent